roles/rsyslog/tasks/main.yml

```diff
@@ -209,12 +209,12 @@
 
       vars:
         __logging_forwards_tls: "{{ logging_outputs | selectattr('type', '==', 'forwards') |
-                                  selectattr('pki', 'defined') |
-                                  selectattr('pki', '==', 'tls') | list }}"
+                                  selectattr('tls', 'defined') |
+                                  selectattr('tls', 'equalto', True) | list }}"
         __logging_remote_tls: "{{ logging_inputs | selectattr('type', '==', 'remote') |
                                   selectattr('tcp_ports', 'defined') |
-                                  selectattr('pki', 'defined') |
-                                  selectattr('pki', '==', 'tls') | list }}"
+                                  selectattr('tls', 'defined') |
+                                  selectattr('tls', 'equalto', True) | list }}"
```

Comment:
use `selectattr('tls', 'true')` - see https://jinja.palletsprojects.com/en/2.10.x/templates/#true

Reply:
@richm "New in version 2.11" - I don't think we want to use it!

Reply:
@pcahyna, @richm, let me replace `true` & `false` with 'equalto', True & False...

Reply:
Opened a pr - https://github.com/linux-system-roles/logging/pull/169

Reply:
sorry I didn't notice this - I guess I assumed since it was on the 2.10 doc page it was supported in 2.10 :-(
but if we are worried about 2.7 support, then this problem goes beyond just `true` and `false` tests being supported